File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/ProductLabels/ProductLabelBasic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Labels;
using Ohtap.ECommerce.Domain.Models.Objects.ProductLabels;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.ProductLabels
{
    public class ProductLabelBasic : IProductLabelData
    {
        public string ID { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public int Order { get; set; }
        public string Description { get; set; }
        public int State { get; set; }
        public string Value { get; set; }
        public ILabelData Label { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/ProductLabels/ProductLabelBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Promotions;



use Ohtap\ECommerce\DomainModels\Objects\Attributes\IAttribute;
use Ohtap\ECommerce\DomainModels\Objects\Attributes\IAttributeData;
use Ohtap\ECommerce\DomainModels\Objects\Discounts\IDiscountData;
use Ohtap\ECommerce\DomainModels\Objects\Labels\ILabelData;

class ProductLabelBasic implements IProductLabelData
{

    public string $Description;
    public ILabelData $Label;
    public string $Value;
    public int $State;
    public int $OrderNo;
    public string $Name;
    public string $ID;

}